src/libraries/Microsoft.PowerFx.Interpreter/Functions/CommonErrors.cs

```diff
@@ -69,6 +69,16 @@ public static ErrorValue OverflowError(IRContext irContext)
             });
         }
 
+        public static ErrorValue BadLanguageCode(IRContext irContext, string languageCode)
+        {
+            return new ErrorValue(irContext, new ExpressionError()
+            {
+                Message = $"Language code {languageCode} not supported",
```

Comment:
As this will happen when the language culture is invalid, i'd name this error BadCulture and fix the error message accordingly ("Invalid Culture {...